crates/holochain/src/conductor/conductor/tests.rs

```diff
@@ -282,6 +283,13 @@ async fn test_uninstall_app() {
         )
         .await;
 
+    // Await integration of both actions.
+    retry_until_timeout!(1_000, 10, {
```

Comment:
The above action is at times not integrated yet when the following "read" wants to get it. All DHT gets require actions to be integrated, so I'm convinced now that we can increase test stability by awaiting integration in cases like this of create with immediate get.